.phpunit.result.cache

```diff
@@ -0,0 +1 @@
681711721685132943093776702880859375;s:71:"tests\phpunit\Precheck\GitCopyAgeTest::testWorkingCopyPastMaxWarningAge";d:0;s:61:"tests\phpunit\Precheck\GitCopyAgeTest::testWorkingCopySuccess";d:0.0109999999999999993616217608405349892564117908477783203125;s:69:"tests\phpunit\Precheck\GitCopyBehindTest::testShouldRunCheckShouldRun";d:0;s:72:"tests\phpunit\Precheck\GitCopyBehindTest::testShouldRunCheckShouldNotRun";d:0;s:70:"tests\phpunit\Precheck\GitCopyBehindTest::testShouldRunCheckNoBranches";d:0;s:68:"tests\phpunit\Precheck\GitCopyBehindTest::testCheckWorkingCopyBehind";d:0;s:71:"tests\phpunit\Precheck\GitCopyBehindTest::testCheckWorkingCopyNotBehind";d:0;s:63:"tests\phpunit\Precheck\GitCopyBehindTest::testCheckShouldNotRun";d:0;s:62:"tests\phpunit\Precheck\GitWarnOnBlocklistedTest::testNoChanges";d:0;s:71:"tests\phpunit\Precheck\GitWarnOnBlocklistedTest::testNoBlockListedFiles";d:0;s:73:"tests\phpunit\Precheck\GitWarnOnBlocklistedTest::testWithBlockListedFiles";d:0;s:72:"tests\phpunit\Precheck\GitWarnOnBlocklistedTest::testWithSafeListedFiles";d:0;s:86:"tests\phpunit\Precheck\GitWarnOnBlocklistedTest::testWithSafeListedAndBlockListedFiles";d:0;s:63:"tests\phpunit\Precheck\GitWarnOnBlocklistedTest::testStagedOnly";d:0;s:64:"tests\phpunit\Precheck\ScriptRunnerTest::testScriptDoesNotExists";d:0;s:68:"tests\phpunit\Precheck\ScriptRunnerTest::testScriptExistsAndSucceeds";d:0;s:65:"tests\phpunit\Precheck\ScriptRunnerTest::testScriptExistsAndFails";d:0;s:68:"tests\phpunit\RepoManager\GitTest::testGenerateDiffStagedSuccessfull";d:0.001000000000000000020816681711721685132943093776702880859375;s:58:"tests\phpunit\RepoManager\GitTest::testGenerateDiffFailure";d:0;s:62:"tests\phpunit\RepoManager\GitTest::testParseLocalBranchSuccess";d:0;s:62:"tests\phpunit\RepoManager\GitTest::testParseLocalBranchFailure";d:0;s:55:"tests\phpunit\RepoManager\GitTest::testGetRemoteSuccess";d:0;s:63:"tests\phpunit\RepoManager\GitTest::testGetRemoteFailWithDefault";d:0;s:61:"tests\phpunit\RepoManager\GitTest::testGetRemoteFailNoDefault";d:0;s:72:"tests\phpunit\RepoManager\GitTest::testGetRemoteBranchFromTrackingConfig";d:0;s:82:"tests\phpunit\RepoManager\GitTest::testGetRemoteBranchNoTrackingRemoteWithSameName";d:0;s:72:"tests\phpunit\RepoManager\GitTest::testGetRemoteBranchNoTrackingNoRemote";d:0;s:46:"tests\phpunit\TryRunner\RunnerTest::testSimple";d:0.00899999999999999931998839741709161899052560329437255859375;s:57:"tests\phpunit\TryRunner\RunnerTest::testHonorRemoteBranch";d:0.00600000000000000012490009027033011079765856266021728515625;s:77:"tests\phpunit\Util\OptionsUtilTest::testParseExtraParameters with data set #0";d:0;s:77:"tests\phpunit\Util\OptionsUtilTest::testParseExtraParameters with data set #1";d:0;s:77:"tests\phpunit\Util\OptionsUtilTest::testParseExtraParameters with data set #2";d:0;s:77:"tests\phpunit\Util\OptionsUtilTest::testParseExtraParameters with data set #3";d:0;s:77:"tests\phpunit\Util\OptionsUtilTest::testParseExtraParameters with data set #4";d:0;}}}
```

Comment:
Can we add this file to _.gitignore_?

Reply:
Yeah, good call! I did not mean to add this to the PR.